adoc/admin-security-ldap.adoc

```diff
@@ -47,72 +45,43 @@ To replace the automatically generated certificate with your own, follow these s
 docker stop 389-ds
 ----
 
-. Copy the external certificate `<rootCA.pem>` to a mounted data directory `<DATA_DIR>/config/`.
+. Copy the external private key `Server-Cert.crt`, certificate `Server-Cert.csr`, and password `pwdfile-import.txt` to a mounted data directory `<DATA_DIR>/config/`.
```

Comment:
I assume Server-Cert.crt is the certificate and not the private key. (crt extension)
I assume Server-Cert.csr is the certificate signing request (csr extension)
-> so are you sure we need crt+crt, here? I would exect to see the crt and key here.

Regarding the pwdfile-import.txt -> what if the key does not have a password set? Is it required to have a password set on the key?